src/sql/parts/profiler/editor/profilerInput.ts

```diff
@@ -130,6 +148,15 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 		this._onColumnsChanged.fire(this.columns);
 	}
 
+	public get connectionName(): string {
```

Comment:
We already have some similar logic for displaying connection names in the `_setConnectionText` method of `connectionStatus.ts`. Maybe there's a way to combine the two?